source/learning-wazuh/hidden-processes.rst

```diff
@@ -85,54 +85,30 @@ Wazuh a very effective Linux rootkit detection application by looking for genera
         and try again. Sometimes it will take several tries to work.
 
 The kernel-level rootkit “Diamorphine” is now installed on this system! By default it is hidden so we
-are not able to detect it by running “lsmod”.  Only with a special "kill" signal can we make Diamorphine
-unhide itself:  Try it out:
+are not able to detect it by running “lsmod”.  Only with a special "kill" signal can we make Diamorphine unhide itself. Try it out:
 
     .. code-block:: console
 
         # lsmod | grep diamorphine
         # kill -63 509
         # lsmod | grep diamorphine
-
-    .. code-block:: none
-        :class: output
-
         diamorphine            13155  0
-
-    .. code-block:: console
-
         # kill -63 509
         # lsmod | grep diamorphine
 
-    .. code-block:: none
-        :class: output
-
-
 
-
-    In the case of Diamorphine, any attempt to send a kill signal ``-63`` to any process whether it exists or not,
-    will toggle whether the Diamorphine kernel module hides itself.
+    In the case of Diamorphine, any attempt to send a kill signal ``-63`` to any process whether it exists or not, will toggle whether the Diamorphine kernel module hides itself.
 
     This rootkit also allows you to hide a selected processes from being seen by the "ps" command for example.
-    Run the following commands to see how the rsyslog process is first visible, then send the ``-31`` signal to
-    its pid and observe how the process is no longer visible
+    Run the following commands to see how the rsyslog process is first visible, then send the ``-31`` signal to its pid and observe how the process is no longer visible.
```

Comment:
When the output is supposed to have an empty response, there should be a clarifying text to help the user understand it.

Regards,

David

Reply:
Hi, I've added a sentence after the command that would return an empty output. Since I'm not an expert on this, maybe it's not clear, so please review it and feel free to change it yourself if you feel like it.


This is how the notes look with the last commit I made:
![Screenshot 2020-03-03 at 12 33 51](https://user-images.githubusercontent.com/16825724/75772169-b6dba600-5d4b-11ea-8901-bddf4e3e6606.png)

![Screenshot 2020-03-03 at 12 33 42](https://user-images.githubusercontent.com/16825724/75772256-e1c5fa00-5d4b-11ea-818a-946f38d4e42b.png)

As I said, I know very little about the content on this page, so feel free to correct me.